src/components/CardSelector/CardSelector.tsx

```diff
@@ -2,16 +2,42 @@ import React from 'react';
 import clsx from 'clsx';
 
 import './CardSelector.scss';
-import type {CardSelectorProps} from './CardSelector.types';
+import type {CardSelectorProps, ThumbnailProps} from './CardSelector.types';
 import {Typography} from '~/components';
 import {FileBroken, Image} from '~/icons/components';
 
+const ThumbnailCmp: React.FC<ThumbnailProps> = ({thumbnail, thumbnailType, thumbnailAlt}) => {
+    console.log(thumbnail);
```

Comment:
might be better use debug instead of log if we want to keep, or remove

Reply:
oups I forgot to remove it